README.md

```diff
@@ -767,6 +767,7 @@ Follows best practices and conventions to provide you a SOLID development experi
 
 ### Tools
 * [CliFx](https://github.com/Tyrrrz/CliFx) - Declarative framework for building command line interfaces.
+* [Typin](https://github.com/adambajguz/Typin) - Simple to use declarative framework for interactive CLI applications and command line tools (direct mode) that has its roots in CliFx.
```

Comment:
please order into alphabet order

Reply:
I don't know how to do this because I have removed my fork ;<

Reply:
Then I will close this PR. Feel free to open it again